events/wc_undead_events.txt

```diff
@@ -1632,101 +1617,321 @@ character_event = {
-	only_playable = yes
-	only_capable = yes
-	prisoner = no
-
-	trigger = {
-		always = no	# TODO
+	
+	immediate = {
+		add_trait = travelling
 		
-		is_adult = yes
-		is_incapable = no
-		prisoner = no
-		is_inaccessible_trigger = no
+		# You feel guilt of killing that character
+		if = {
+			limit = { any_kill_list_character = { is_close_relative = ROOT } }
+			random_kill_list_character = {
+				limit = { is_close_relative = ROOT }
+				save_event_target_as = target_killed_character
+			}
+		}
+	}
 
-		e_scourge = {
-			holder_scope = {
-				trait = creature_wraith
-				trait = lich_king
-				OR = {
-					has_character_modifier = mind_control_3
-					has_character_modifier = mind_control_4
-				}
+	option = { #For the champion
+		name = EVTOPTA_WCUND_400
+		
+		# If you finished the Champion event chain, you get +1 event
+		if = {
+			limit = { has_character_flag = finished_champion_event_chain_flag }
+			73 = { province_event = { id = WCUND.410 days = 30 } }
+		}
+		else = {
+			73 = { province_event = { id = WCUND.524 days = 60 } }
+		}
+	}
+}
+### Return to Northrend
+# Delayed event
+province_event = {
+	id = WCUND.410
+	
+	is_triggered_only = yes
+	hide_window = yes
+	
+	immediate = {
+		# If... these conditions are met, event chain can go on
+		if = {
+			limit = {
+				event_target:target_lich_king = { merging_lich_king_trigger = yes }
+				event_target:target_champion = { merging_champion_trigger = yes }
 			}
+			event_target:target_champion = { narrative_event = { id = WCUND.411 } }
+		}
+		# Else... notify that event chain failed
+		else = {
+			event_target:target_lich_king = { character_event = { id = WCUND.540 } }
+			event_target:target_champion = { character_event = { id = WCUND.540 } }
 		}
 	}
+}
+# Delayer event
+narrative_event = {
+	id = WCUND.411
+	title = EVTTITLE_WCUND_400
+	desc = EVTDESC_WCUND_411
+	picture = GFX_evt_icecrown
+	border = GFX_event_narrative_frame_intrigue
 
+	is_triggered_only = yes
+	
+	option = {
+		name = EVTOPTA_WCUND_411
+		
+		remove_slothful_or_add_diligent_silently_effect = yes
+		
+		73 = { province_event = { id = WCUND.524 days = 30 } }
+	}
+}
+### Merges with Lich King
+# Delayer event
+province_event = {
+	id = WCUND.524
+	
+	is_triggered_only = yes
+	hide_window = yes
+	
 	immediate = {
-		e_scourge = { holder_scope = { save_event_target_as = target_lich_king } }
-		set_global_flag = merging_with_the_lich_king
+		# If... these conditions are met, event chain can go on
+		if = {
+			limit = {
+				event_target:target_lich_king = { merging_lich_king_trigger = yes }
+				event_target:target_champion = { merging_champion_trigger = yes }
+			}
+			event_target:target_champion = { narrative_event = { id = WCUND.525 } }
+		}
+		# Else... notify that event chain failed
+		else = {
+			event_target:target_lich_king = { character_event = { id = WCUND.540 } }
+			event_target:target_champion = { character_event = { id = WCUND.540 } }
+		}
 	}
+}
+# Delayed event
+narrative_event = {
+	id = WCUND.525
```

Comment:
No character?
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46576860/73928184-b7675500-48e3-11ea-9bc5-71071e10b52c.png)

</details>

Reply:
Fixed https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/commit/ae3d84fe8cfc39a8f6c05ed620148ae813d7ea5c.